R/as_model_out_df.R

```diff
@@ -0,0 +1,211 @@
+#' Convert model output to a `as_model_out_tbl` class object.
+#'
+#' @param tbl a `data.frame` or `tibble` of model output data returned from a query
+#' to a `<hub_connection>` object.
+#' @param model_id_col character string. If an `model_id` column does not
```

Comment:
```suggestion
#' @param model_id_col character string. If a `model_id` column does not
```

Reply:
Fixed in 45d3c0da56e26292c93ff8198fa310184f2239d5 👍